stores/sql/sqlite/main.go

```diff
@@ -25,66 +27,72 @@ func NewMainDatabase(db *dsql.DB, log *zap.SugaredLogger, lqd, ltd time.Duration
 	}
 }
 
+func (b *MainDatabase) ApplyMigration(fn func(tx sql.Tx) (bool, error)) error {
+	return applyMigration(b.db, fn)
+}
+
 func (b *MainDatabase) Close() error {
 	return b.db.Close()
 }
 
-func (b *MainDatabase) Version(_ context.Context) (string, string, error) {
-	return version(b.db)
+func (b *MainDatabase) DB() *sql.DB {
+	return b.db
+}
+
+func (b *MainDatabase) CreateMigrationTable() error {
+	return createMigrationTable(b.db)
+}
+
+func (b *MainDatabase) MakeDirsForPath(tx sql.Tx, path string) (uint, error) {
```

Comment:
same remarks, might want to deduplicate this logic

Reply:
The `makeDirsForPath` function will be removed as soon as the calling methods rely on the raw backend. Then it's only this one and the `MySQL` implementation which are the same right now but I'll probably f/u with a change to use `RETURNING` in the SQLite one.